dark_emulator/darkemu/de_interface.py

```diff
@@ -61,7 +61,7 @@ def set_cosmology(self, cparam):
 
         Args:
             cparam (numpy array): Cosmological parameters
-                :math:`(\omega_b, \omega_{m}, \Omega_{de}, \ln(10^{10}A_s), n_s, w)`
+                :math:`(\omega_b, \omega_{c}, \Omega_{de}, \ln(10^{10}A_s), n_s, w)`
```

Comment:
Sorry this might be picky, but it would be nice to change `\omega_{c}` to `\omega_c`.

Reply:
done!